README.md

```diff
@@ -634,6 +634,15 @@ ZenHub is the first and only project management suite that works natively within
  ![ZenHub](https://dl.dropboxusercontent.com/s/yosmyg8zsl5tyc5/zenhub.jpg)
 </details>
 
+## asciinema Player for GitHub <a href="https://chrome.google.com/webstore/detail/asciinema-player-for-gith/mkllmpcfhjcbkmegdpnaemhpdddffhil"><img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" width="24" /></a>
```

Comment:
Please wrap the title with a link to your repo. That way people can easily find the source. Thanks

Reply:
Sure thing, done :)

Reply:
Thank you